moto/s3/models.py

```diff
@@ -676,6 +678,8 @@ def get_bucket_website_configuration(self, bucket_name):
 
     def set_key(self, bucket_name, key_name, value, storage=None, etag=None):
         key_name = clean_key_name(key_name)
+        if storage is not None and not(storage in ["STANDARD", "REDUCED_REDUNDANCY", "STANDARD_IA", "ONEZONE_IA"]):
```

Comment:
I think
```python
if storage is not None and storage not in ("STANDARD", "REDUCED_REDUNDANCY", "STANDARD_IA", "ONEZONE_IA")):
```
Looks better :wink: though functionally equivalent, same for line 71.

Would it not make sense to move the list of storage classes to an allcaps variable at the top of the file, then everywhere can reference one copy